### PR TITLE
fix(schema): add stop arg to schema functions

### PR DESCRIPTION
**Pull request by nathanielc**
Previously the `stop` argument could not be added to the schema
functions because the default value would overly constrain the arguments
type. That has been fixed.

Note sure how to add tests for this change as we don't actually have
tests for calling the functions rather just for the push downs matching.

### Done checklist
- [ ] docs/SPEC.md updated
- [ ] Test cases written

### CODE REVIEW

**Comment by nathanielc** [comment]
@jsternberg Thoughts on how I could testthis?

File: stdlib/influxdata/influxdb/schema/schema.flux [hunk 1]
```diff
@@ -240,4 +268,4 @@ measurementFieldKeys = (bucket, measurement, start=-30d) =>
 //
 // tags: metadata
 //
-measurements = (bucket) => tagValues(bucket: bucket, tag: "_measurement")
+measurements = (bucket, start=_startDefault, stop=_stopDefault) => tagValues(bucket: bucket, tag: "_measurement", start: start ,stop: stop)
```
Comment (jsternberg) [inline stdlib/influxdata/influxdb/schema/schema.flux:271]: A type in the signature here.

**Review by jsternberg — APPROVED** [review]

**Comment by nathanielc** [comment]
We have tests but they just mirror the function bodies instead of calling the functions directly. This is to get around the fact that the functions call `from()` directly.

**Comment by Marwes** [comment]
I guess we could add `from` as a default argument to the functions, and pass in a test `from` function for testing. Or add private variants of all functions under test that accepts `from` as a parameter, then invoke those from the current

```
tagValues = (
    bucket,
    tag,
    predicate=(r) => true,
    start=_startDefault,
    stop=_stopDefault,
)  => tagValuesInner(bucket, tag, predicate, start, stop, from)

tagValuesInner = (
    bucket,
    tag,
    predicate=(r) => true,
    start=_startDefault,
    stop=_stopDefault,
    from,
) =>
    from(bucket: bucket)
        |> range(start: start, stop: stop)
        |> filter(fn: predicate)
        |> keep(columns: [tag])
        |> group()
|> distinct(column: tag)
```

**Comment by nathanielc** [comment]
@Marwes Thanks for the idea, it decided to implement something very similar by using an `option`. 

As a result I did a complete rewrite of the tests but they are much easier to understand now.

**Comment by nathanielc** [comment]
Also I have run these tests against InfluxDB OSS and they pass.

**Review by noramullen1 — APPROVED** [review]

**Review by Marwes — APPROVED** [review]

**Pull request merged**